packages/@aws-cdk/tmp-toolkit-helpers/test/api/diff/diff.test.ts

```diff
@@ -265,59 +260,4 @@ describe('formatSecurityDiff', () => {
       '(NOTE: There may be security-related changes not in this list. See https://github.com/aws/aws-cdk/issues/1299)',
     );
   });
-
```

Comment:
these tests don't make sense now that `tmp-toolkit-helpers` does not deal with approval levels at all